docs/overview/intro.md

```diff
@@ -19,17 +19,14 @@ Whether you're building a simple API or a complex enterprise application, Expres
 
 ## Installing the CLI
 
-Before creating a project make sure you have ExpressoTS **[CLI](../cli/overview.md)** installed globally on your machine.
-
-```bash
-npm i -g @expressots/cli
-```
+To start a new ExpressoTS project, you have two options: create a new project using [ExpressoTS CLI](../cli/overview.md) or clone the initial project from GitHub. While both methods produce the same result, we recommend using the Expresso CLI to create a new project as it offers a more streamlined experience.
 
 ## Creating a new project
 
 To create a project using the CLI, simply run the following commands. The CLI will prompt you with a few questions and then create a new project directory with the name you provide, along with the @expressots/core package and a starter project structure.
 
 ```bash
+npm i -g @expressots/cli
```

Comment:
this live of installation can be removed as it was before as is being described in the Installing the CLI section

## Installing the CLI
Before creating a project make sure you have ExpressoTS CLI installed globally on your machine.

npm i -g @expressots/cli